subscriptions/drpolicy.yaml

```diff
@@ -4,6 +4,9 @@ kind: DRPolicy
 metadata:
   name: dr-policy
 spec:
-  clusterNames:
-  - hub
-  - cluster1
+  drClusterSet:
+  - name: hub
+    s3ProfileName: ""
```

Comment:
I suggest setting the s3ProfileName to minio-on-hub to match the e2e setup, like [here](https://github.com/vdeenadh/ocm-ramen-samples/blob/s3/subscriptions/drpolicy.yaml).

Reply:
Updated.  Thank you Veera!